File: isomorphic-strings/IsomorphicString.c
bool isIsomorphic(char * s, char * t){
  char arr1[256]={'\0'};
  char arr2[256]={'\0'};

  while(*s !='\0'){
    if(arr1[*s] == '\0')
      arr1[*s] = *t;
    
    if(arr2[*t] == '\0')
      arr2[*t] = *s;

    if( arr1[*s] != *t || arr2[*t] != *s)
      return false;
    s++;
    t++;
  }
  return true;
}

File: isomorphic-strings/IsomorphicString.kt
class Solution {
    fun isIsomorphic(s: String, t: String): Boolean {
        var arr1 = CharArray(256)
        var arr2 = CharArray(256)

        for(i in 0..s.length-1){
            var charInS: Int = s[i].toInt()
            var charInT: Int = t[i].toInt()

            if(arr1[charInS] =='\u0000')
                arr1[charInS] = t[i] 
            if(arr2[charInT] == '\u0000')
                arr2[charInT] = s[i] 

            if(arr1[charInS] != t[i] || arr2[charInT] != s[i]){
                return false;
            }
        }
        return true
    }
}